src/components/coursemanage/grading/edit-submission.tsx

```diff
@@ -234,12 +237,21 @@ export const EditSubmission = () => {
       >
         <DialogTitle id="alert-dialog-title">{'Logs'}</DialogTitle>
         <DialogContent>
-          <Typography
-            id="alert-dialog-description"
-            sx={{ fontSize: 10, fontFamily: "'Roboto Mono', monospace" }}
-          >
-            {logs}
-          </Typography>
+          {logs ? (
+            <Typography
+              id="alert-dialog-description"
+              sx={{ fontSize: 10, fontFamily: "'Roboto Mono', monospace" }}
+            >
+              {logs}
```

Comment:
Könnte man umbschrieben in `{logs | "No logs available"}` dann spart man sich den zweiten Typography tag